server/channels/app/platform/config.go

```diff
@@ -42,17 +42,17 @@ func (ps *PlatformService) Config() *model.Config {
 }
 
 // getSanitizedConfig gets the configuration without any secrets.
-func (ps *PlatformService) getSanitizedConfig(rctx request.CTX) *model.Config {
+func (ps *PlatformService) getSanitizedConfig(rctx request.CTX, opts *model.SanitizeOptions) *model.Config {
```

Comment:
Is there a strong reason to use a pointer for opts rather than passing it by value? Passing by value often simplifies usage, especially since default values are handled automatically.